### ISSUE TITLE
Change in plugin loading with Vagrant after 2.13 stops VM building

### ISSUE THREAD

**Issue by Chimoe** [Submitty/Submitty#2830]
I failed multiple times on plugin test when I tried to run `vagrant up` on my machine this afternoon.

It seems this section of the [Vagrantfile](https://github.com/Submitty/Submitty/blob/master/Vagrantfile) always return false.

```ruby
unless Vagrant.has_plugin?('vagrant-vbguest')
  raise 'vagrant-vbguest is not installed!'
end
```

After I downgrade my Vagrant from 2.14 to 2.12 I passed the plugin test.

Possible workarounds can be found here [hashicorp/vagrant#10161](https://github.com/hashicorp/vagrant/issues/10161#issuecomment-417701063).

**Comment by MasterOdin** [comment]
There's no real need for us to make any changes for now as hashicorp/vagrant#10165 was merged so once 2.15 gets released, this code block will go back to working just fine. For now, no one should upgrade past 2.12 though as you correctly note.

**Comment by MasterOdin** [comment]
Vagrant 2.15 was released which fixes this bug. It's recommended anyone that was on 2.13 or 2.14 upgrade to that version now.
